File: app/src/main/java/com/rr/rhythms/Entities/People.java
package com.rr.rhythms.Entities;

import com.rr.rhythms.Interfaces.IPerson;

/**
 * Created by Huruk on 7/9/2016.
 */
public class People implements IPerson {
    private Person[] _people;

    public People(Person[] person) {
        _people = person;
    }

    public Person getAt(int i) {
        return _people[i];
    }

    public Person[] getPeople() {
        return _people;
    }
}

File: app/src/main/java/com/rr/rhythms/Entities/People.kt
package com.rr.rhythms.Entities

import com.rr.rhythms.Interfaces.IPerson

/**
 * Created by Huruk on 7/9/2016.
 */
class People(val people: Array<Person>) : IPerson {
    fun getAt(i: Int): Person {
        return people[i]
    }
}
